cli/command_policy_import.go

```diff
@@ -0,0 +1,91 @@
+package cli
+
+import (
+	"bufio"
+	"context"
+	"encoding/json"
+	"io"
+	"os"
+
+	"github.com/kopia/kopia/repo"
+	"github.com/kopia/kopia/snapshot"
+	"github.com/kopia/kopia/snapshot/policy"
+	"github.com/pkg/errors"
+)
+
+type commandPolicyImport struct {
+	policyTargetFlags
+	filePath string
+
+	stdin io.Reader
+}
+
+func (c *commandPolicyImport) setup(svc appServices, parent commandParent) {
+	cmd := parent.Command("import", "Imports policies from a specified file, or stdin if no file is specified.")
+	cmd.Flag("from-file", "File path to import from").StringVar(&c.filePath)
+
+	c.policyTargetFlags.setup(cmd)
+	c.stdin = svc.stdin()
```

Comment:
don't invoke method on `svc` in the `setup` instead of this, store `svc` in the struct and invoke `stdin()` in the `run()` method.